index.bs

```diff
@@ -203,7 +201,7 @@ Unlike many video standards, AV1 does not distinguish the display order from the
 
 When a temporal unit contains more than one frame, the sample corresponding to that temporal unit should be marked using the [=AV1MultiFrameSampleGroupEntry=].
 
-The grouping_type_parameter is not defined for the SampleToGroupBox with grouping types defined in this specification (i.e. 'av1m', 'av1s', 'av1M').
+The grouping_type_parameter is not defined for the SampleToGroupBox with grouping types defined in this specification (i.e. 'av1m', 'av1s', 'av1M', 'av1f').
```

Comment:
Unrelated change?

Reply:
No it is. The 4CC for the new sample group is av1f. Maybe I should simply remove the text between parenthesis. That would avoid updating it every time a new Sample Group is defined ...

Reply:
Oh. Somehow I misparsed it as related to AVIF-the-image-container.